--- flake8-3.7.8/setup.py.orig	2019-07-08 20:05:23.000000000 +0200
+++ flake8-3.7.8/setup.py	2019-10-22 22:07:31.791565818 +0200
@@ -19,7 +19,7 @@
     # http://flake8.pycqa.org/en/latest/faq.html#why-does-flake8-use-ranges-for-its-dependencies
     # And in which releases we will update those ranges here:
     # http://flake8.pycqa.org/en/latest/internal/releases.html#releasing-flake8
-    "entrypoints >= 0.3.0, < 0.4.0",
+    "entrypoints >= 0.3, < 0.4",
     "pyflakes >= 2.1.0, < 2.2.0",
     "pycodestyle >= 2.5.0, < 2.6.0",
     "mccabe >= 0.6.0, < 0.7.0",
